tests/functional/object/mcg/ui/test_mcg_ui.py

```diff
@@ -441,3 +442,181 @@ def test_bucket_create(self, setup_ui_class_factory):
         assert (
             bucket_ui.create_folder_in_bucket()
         ), "Failed to create and upload folder in bucket"
+
+    def test_bucket_delete(self, setup_ui_class_factory):
```

Comment:
I think we need to add tearDown function that will clean up cluster of created resources for the next tests. 
Also we risk to leave these buckets not being destroyed upon cluster-destroy job.